bookchain-miner/src/storage.cpp

```diff
@@ -18,7 +18,7 @@ void purgeChain(const std::string& filename) {
     blockchainFile.close();
 }
 
-void dumpChain(std::vector<Bloock> bloockChain, const std::string& filename) {
+void dumpChain(std::vector<Bloock>& bloockChain, const std::string& filename) {
```

Comment:
Do we need a function to load a blockchain from file?

Reply:
We might, and it's already in there: `std::vector<Bloock> getChain(const std::string& filename)`